tests/vmi_multus_test.go

```diff
@@ -1048,3 +1054,38 @@ func configureNodeNetwork(virtClient kubecli.KubevirtClient) {
 		return int(daemonSet.Status.NumberAvailable)
 	}, time.Minute, time.Second).Should(Equal(len(nodes.Items)))
 }
+
+func checkSriovEnabled(virtClient kubecli.KubevirtClient, sriovResourceName string) bool {
+	nodes := tests.GetAllSchedulableNodes(virtClient)
+	Expect(nodes.Items).ToNot(BeEmpty(), "There should be some compute node")
+
+	for _, node := range nodes.Items {
+		var sriovallocate bool = false
+		var sriovcapacity bool = false
+		resourceList := node.Status.Allocatable
+		for k, v := range resourceList {
+			if string(k) ==  sriovResourceName {
+				if v.Value() > 0 {
+					sriovallocate = true
+					break
+				}
+			}
+		}
+		if !sriovallocate{
+			return false
+		}
+		capacityList := node.Status.Capacity
```

Comment:
I think it is enough to check only for allocate. It should never be lower than allocatable